src/styles/pages/_path.scss

```diff
@@ -70,3 +70,10 @@
   margin: 0;
   padding: 0;
 }
+
+.w-path-link--external::after {
+  @include font-material-icon();
+  content: 'launch';
+  display: inline-block;
+  vertical-align: middle;
+}
```

Comment:
@robdodson logic for styling a link as external

Reply:
I think this seems fine. There are somewhat more reusable ways of doing it:

```
a[href^="https://"]:not([href*="mysite.com"]) { ... }
```

but I'm unsure what the affects would be across the site if we did that. I'd like to plan a bigger CSS refactor of the site at some point so for now I think scoping this change down to this one use case is a good approach.